src/core/Expectation.js

```diff
@@ -1,10 +1,4 @@
 getJasmineRequireObj().Expectation = function(j$) {
-  var promiseForMessage = {
-    jasmineToString: function() {
-      return 'a promise';
```

Comment:
Since the `actual` isn't always going to be a Promise there is no need for this to override `actual` in the default message.